### DISCUSSION TITLE
Converting string to number

### DISCUSSION THREAD

**Discussion by PEWOLO** [typst/typst#3225] [General]
Hey everyone,

Is there a way to convert string to number in Typst?

**Comment by LaurenzV** [comment]
You can do it like this:

`#int("34")`

But by the way, the "Issues" section is meant to be used for bug reports, not for Q&A. If you have specific questions about how to use Typst, please ask them in [Discussions](https://github.com/typst/typst/discussions/) or on the Discord server.

Thanks. :)
